src/index.html

```diff
@@ -7,8 +7,198 @@
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
+
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <nav>
+        <button id="home">HOME</button
+        ><button id="download">DOWNLOAD MY FULL RESUME</button>
+      </nav>
+      <div id="top">
+        <div id="shade1">
+          <div id="name_title">John Roberts</div>
+          <div id="name_subtitle">BUSINESS EXPERT</div>
+          <div id="portrait"><img src="" alt="John Roberts" /></div>
+        </div>
+        <div id="shade2"><img src="" alt="backgroud_PIC" /></div>
+      </div>
+      <div id="pers">
+        <ul>
+          <li>
+            <div id="am_section">
+              <div id="am_t">ABOUT ME</div>
+              <div id="am_social">
+                <div id="linki_top" class="socialM">
+                  <a href="https://www.linkedin.com/"
+                    ><img src="" alt="LinkIn"
+                  /></a>
+                </div>
+                <div id="twitt_top" class="socialM">
+                  <a href="https://www.twitter.com/"
+                    ><img src="" alt="Twitter"
+                  /></a>
+                </div>
+                <div id="faceb_top" class="socialM">
+                  <a href="https://www.facebook.com/"
+                    ><img src="" alt="Facebook"
+                  /></a>
+                </div>
+              </div>
+              <hr />
+              <div id="am_p"></div>
+            </div>
+          </li>
+          <li>
+            <div id="expr_section">
```

Comment:
Just a suggestion: for more readability, id and class definitions should be more meaningful